src/App.tsx

```diff
@@ -61,8 +77,24 @@ function Welcome(_props: RouteComponentProps): JSX.Element {
   return (
     <div css={c.welcome}>
       <Hello compiler="TypeScript" framework="React" />
-      <button onClick={() => navigate('/')}>Home</button>
       <Search />
+      <Button variant="contained" color="primary" onClick={() => navigate('/')}>
+        Home
+      </Button>
+      <Copyright />
+    </div>
+  );
+}
+
+function SearchForm(_props: RouteComponentProps): JSX.Element {
+  return (
+    <div css={c.search}>
+      <SearchRecords />
```

Comment:
Oh, I see. You did what I suggested above, you just used different names. Good! (I still think `Form` sounds embeddable, not like a top-level page, though.)